lib/statsd/instrument/rubocop/positional_arguments.rb

```diff
@@ -0,0 +1,60 @@
+# frozen-string-literal: true
+
+module RuboCop
+  module Cop
+    module StatsD
+      class PositionalArguments < Cop
+        MSG = 'Use keyword arguments for StatsD calls'
+
+        STATSD_SINGLETON_METHODS = %i{increment gauge measure set histogram distribution key_value}
+        NON_POSITIONAL_SECOND_ARGUMENT_TYPES = [:block_pass, :hash]
+
+        def on_send(node)
+          if node.receiver&.type == :const && node.receiver&.const_name == "StatsD"
+            if STATSD_SINGLETON_METHODS.include?(node.method_name)
+              arguments = node.arguments
+              if arguments.length > 2 && !NON_POSITIONAL_SECOND_ARGUMENT_TYPES.include?(arguments[2].type)
```

Comment:
What if the options hash is passed using a local variable or through a keyword splat (e.g. `**kwargs`)?

Reply:
I've been testing this cop on Shopify/shopify. 

`**kwargs` seem to work fine.

However, when a method is provided, the result is ambiguous, because we don't know whether this method will return a hash (interpreted as keyword arguments) or a sample rate.

This means we can detect these cases, but we'll need to manually rewrite them.